scripts/oncotree_to_oncotree.py

```diff
@@ -28,6 +28,7 @@
 VERSION_API_IDENTIFIER_FIELD = "api_identifier"
 VERSION_RELEASE_DATE_FIELD = "release_date"
 METADATA_HEADER_PREFIX = "#"
+PASSTHROUGH_ONCOTREE_CODE_LIST = ["NA"] # These codes will be passed through the converter without examination or alteration
```

Comment:
@sheridancbio why not empty values or `N/A` anymore? Those are often present in clinical files as well